### PR TITLE
Added Infinity Scroll project

### PR DESCRIPTION
**Pull request by skyHunter-18**
Added a new project called Infinity Scroll which is made by using HTML, CSS and JS.
This project uses Unsplash API to fetch images and this can be hosted on Github pages as static website.

PS: Please add the Hacktoberfest label.

### CODE REVIEW

**Comment by iamwatchdogs** [comment]
@skyHunter-18 Sorry about the linting issue, we have been working on it. We have recently updated the lining configs, can you add a small comment to anyone of your js files to trigger the lining work again.

File: quote-generator/index.md
Comment (iamwatchdogs) [inline quote-generator/index.md:1]: Please rename the `index.md` to `README.md` as it might cause issue while deploying onto GitHub Pages.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]

File: quote-generator/index.md
Comment (skyHunter-18) [reply quote-generator/index.md:1]: Done the changes.

**Review by iamwatchdogs — APPROVED** [review]
Requested changes have been made. Reviewed and Approved by @iamwatchdogs.

**Comment by iamwatchdogs** [comment]
Thank you for contributing, @skyHunter-18. Please check your contribution on [GitHub Pages](https://grow-with-open-source.github.io/Javascript-Projects/).

**Pull request merged**